src/get-started/quickstart.md

```diff
@@ -74,8 +88,9 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
Wouldn't this be a simple pipeline option with hostname/port only to point to an already configured cluster? (How to set up such a cluster would be overload for sure.)